### PR TITLE
Gh 1268 switch safes

### PR DESCRIPTION
**Pull request by sche**
image

### CODE REVIEW

**Review by MouazAlzahabi — APPROVED** [review]

File: MultisigIntegrationTests/CoreData/NetworkTests.swift [hunk 1]
```diff
@@ -197,6 +197,40 @@ class NetworkTests: CoreDataTestCase {
             themeBackgroundColor: "#000000")
     }
 
+    func test_networkSafes() throws {
```
Comment (DmitryBespalov) [inline MultisigIntegrationTests/CoreData/NetworkTests.swift:200]: Missing test cases:
- when no safes exist, then should return empty
- when all safes in the same network, should do the correct ordering

Maybe unrelated, but it's good if we fix it here:
- when multiple safes with different network and same address exist, then works fine
- when last safe removed from the network, then selects next safe in order

File: Multisig/UI/Safe Management/SwitchSafesViewController/SwitchSafesViewController.swift [hunk 2]
```diff
@@ -11,16 +11,20 @@ import UIKit
 final class SwitchSafesViewController: UITableViewController {
     var notificationCenter = NotificationCenter.default
 
-    private var safes = [Safe]()
-    private let addSafeRowIndex = 0
+    private var networkSafes = Network.NetworkSafes()
+    private let addSafeSection = 0
```
Comment (DmitryBespalov) [inline Multisig/UI/Safe Management/SwitchSafesViewController/SwitchSafesViewController.swift:15]: I would prefer a different design - with enums, but OK, no need to rewrite now

**Review by DmitryBespalov — COMMENTED** [review]
tests missing

File: MultisigIntegrationTests/CoreData/NetworkTests.swift (hunk 1 above)
Comment (sche) [reply MultisigIntegrationTests/CoreData/NetworkTests.swift:200]: > when no safes exist, then should return empty
added 

> when all safes in the same network, should do the correct ordering
this is already covered

> when multiple safes with different network and same address exist, then works fine
added

> when last safe removed from the network, then selects next safe in order
I will add it in other already open PR

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**